src/js/components/ShabadControls.js

```diff
@@ -190,12 +205,20 @@ export default class ShabadControls extends React.PureComponent {
                 type="range"
                 min="5"
                 max="50"
-                defaultValue={fontSize * 10}
+                value={fontSize * 10}
                 onChange={e => setFontSize(e.currentTarget.value / 10)}
                 onInput={e => setFontSize(e.currentTarget.value / 10)}
               />
               <big className="gurbani-font">A</big>
             </div>
+            <div className="display-reset-option-content">
+              <a
+                className={`display-option-toggle ${darkMode ? 'active' : ''}`}
+                onClick={resetFontOptions}
+              >
+                Reset
```

Comment:
Could you please use TEXTS from constants file? It'll help us in internationalizing later. `TEXTS.RESET`